tools/esp8266/app.cpp

```diff
@@ -86,37 +86,47 @@ void app::loop(void) {
 
     yield();
 
-    if(checkTicker(&mRxTicker, 5)) {
+    if(checkTicker(&mRxTicker, 5)) 
+    {
```

Comment:
ich mag die geschweiften Klammern nicht in der nächsten Zeile, das bläht den code nur unnötig auf. Ich habe gerade über die Autoformatierung in VSCode gelesen. Automatisch würde gehen, aber da muss jeder bei sich Einstellungen vornehmen - also unpraktisch.
Ich wäre dafür, dass wir Coding-Guidelines erstellen, dann formatiert jeder gleich

Reply:
hier habe ich mal angefangen:
https://github.com/lumapu/ahoy/wiki/Coding-Guidlines

Reply:
Moin @lumapu, ja das ist besser. Danke für deine Mühe. 
Werde ich Zuhause dann wieder "zurück" machen. 

Ich schau mal ob man es via VSCode automatisch realisieren kann, das die Einstellungen aus dem Projekt gezogen werden können.

Reply:
Super, bin gespannt ob du was findest. Ich habe gestern kurz geschaut aber das waren alles manuelle und lokale Einstellungen.
Evtl kann man mit einem Python Script für Einstellung 'von hinten durch die Brust ins Auge' einstellen

Reply:
Schau mal hier: https://code.visualstudio.com/docs/getstarted/settings#_workspace-settingsjson-location